src/api/routers/participantsRouter.ts

```diff
@@ -180,6 +180,34 @@ export function createParticipantsRouter() {
     }
   );
 
+  const updateParticipantParser = z.object({ apiRoles: z.array(z.number()) });
+
+  participantsRouter.put(
+    '/:participantId',
+    isApproverCheck,
+    async (req: ParticipantRequest, res: Response) => {
+      const { participant } = req;
+
+      if (!participant) {
+        return res.status(400).send('Unable to find participant');
```

Comment:
should this be 404? On another note, a bunch of our other endpoints in here don't do this check, they just use `participant!`. Do you think we should do this check more often, or not at all?

Reply:
I don't think it's needed for most as the checkParticipantId usually guarantees a participant I believe. I think though for routes not using it, and instead using `isApprovedCheck` they should probably has the check as I don't think there's a guarantee of participant existing.

Reply:
fair enough. Shouldn't this still be a 404 though?

Reply:
Yeah it would